src/userRepository.js

```diff
@@ -1,9 +1,17 @@
+/* eslint-disable max-len */
 class UserRepository {
-    constructor(userData) {
-    if(userData) {
-      this.userData = userData;
-      this.users = [];
-    }
-    }
+  constructor(userData) {
+    this.userData = this.checkIfDataIsArray(userData)
+    this.users = [];
   }
-  export default UserRepository;
+
+  checkIfDataIsArray(data) {
+    return data instanceof Array ? data : "Error, data for user\'s cannot be found."
```

Comment:
I have never seen this before - and I love it.  This is great for error handling.  Will update the pantry test - this is great for sad paths - it stops it right then and there.